src/vuestic-theme/vuestic-components/vuestic-tabs/VaTabs.demo.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
Should have scroll at least.

![image](https://user-images.githubusercontent.com/5394573/54204049-80769c80-44e4-11e9-8de0-92b88809f4cd.png)